include/sparrow/data_traits.hpp

```diff
@@ -117,6 +123,14 @@ namespace sparrow
         using default_layout = variable_size_binary_layout<value_type, std::string_view, const std::string_view>;  // FIXME: this is incorrect, change when we have the right types
     };
 
+    template <>
+    struct arrow_traits<std::string_view>
```

Comment:
Will we actually store `string_view` in array datas?

Reply:
Removed

Reply:
Looks like it's still not removed?